BaseApp/COD1200.TXT

```diff
@@ -0,0 +1,97 @@
+OBJECT Codeunit 1200 Import Bank Statement
```

Comment:
You've got COD1200 in here twice. This PR remains strange :-) I'll try to see if I can get to the bottom of this.